che-plugins.yaml

```diff
@@ -130,7 +130,7 @@ plugins:
           - /go/bin/che-machine-exec
           - '--url'
           - '127.0.0.1:4444'
-  - id: eclipse/che-machine-exec-plugin/7.25.0
+  - id: eclipse/che-machine-exec-plugin/7.25.1
```

Comment:
Something is wrong, the ID for this plugin was updated to `7.25.1` but the image still points to `7.25.0`